src/modules/user/service.js

```diff
@@ -37,8 +47,31 @@ const getUserById = (id) => {
   return db.User.findByPk(id);
 };
 
+const validatePhoneNumber = (value, helpers) => {
+  const condition = value.match(
+    `^\\+(${ALLOWED_PHONE_NUMBER_COUNTRY_CODE.join('|')})[0-9]{6,10}$`,
+  );
+  return condition ? value : helpers.message(PHONE_NUMBER_IS_INVALID_MESSAGE);
+};
+
+const updateUser = async (user, newData) => {
+  try {
+    const newUser = await user.update({ ...newData });
```

Comment:
Better to use `db.User` instead of `user` instance

Reply:
If we replace it with `const newUser = await db.User.update({ ...newData }, { where: { id: user.id } });`, it does not return an updated user object. I didn't found a way to get an updated user without extra db request, because ["returning" option is Postgres only](https://sequelize.org/master/class/lib/model.js~Model.html#static-method-update). So, we will need to do an extra db request while using `db.User` instead of `user` instance, i think it's worse, isn't it?